packages/image_picker/image_picker/README.md

```diff
@@ -55,7 +55,7 @@ import 'package:image_picker/image_picker.dart';
 
 ### Handling MainActivity destruction on Android
 
-Android system -- although very rarely -- sometimes kills the MainActivity after the image_picker finishes. When this happens, we lost the data selected from the image_picker. You can use `retrieveLostData` to retrieve the lost data in this situation. For example:
+When under high memory pressure the Android system -- although very rarely -- sometimes kills the MainActivity of the application using the image_picker. On Android the image_picker plugin makes use of the default `Intent.ACTOIN_GET_CONTENT` or `MediaStore.ACTION_IMAGE_CAPTURE` intents. This means that while the intent is executing the source application is moved to the background and becomes eligable for cleanup when the system is low on memory. When the intent finished executing, Android will restart the source application, however when this happens, we lost the data selected from the image_picker. In this situation it is possible to use the `ImagePicker.retrieveLostData()` method to retrieve the lost data. For example:
```

Comment:
I think we should probably remove the "although very rarely" since on some lower-end devices it seems pretty common based on issue reports. So:
"When under high memory pressure the Android system may the MainActivity of the application using `image_picker`."

Reply:
Fixed.